controllers/metal3remediation_controller.go

```diff
@@ -222,6 +250,203 @@ func (r *Metal3RemediationReconciler) reconcileDelete(ctx context.Context,
 	return ctrl.Result{}, nil
 }
 
+// remediateRebootStrategy executes the remediation using the reboot strategy.
+// Returns nil, nil when reconcile can continue.
+// Return a Result and optionally an error when reconcile should return.
+func (r *Metal3RemediationReconciler) remediateRebootStrategy(ctx context.Context,
+	remediationMgr baremetal.RemediationManagerInterface, node *corev1.Node) (ctrl.Result, error) {
+	switch remediationMgr.GetRemediationPhase() {
```

Comment:
This is only called from the Running phase, what's the point of checking it at runtime?

Reply:
left over from old version, removed, thx